geode-core/src/main/java/org/apache/geode/internal/tcp/Connection.java

```diff
@@ -1322,6 +1328,14 @@ private void createBatchSendBuffer() {
     this.batchFlusher.start();
   }
 
+  public void onIdleCancel() {
```

Comment:
The name of this function is not intuitive to me. Could this be named `removeIdleReceiver` or something? Or just `removeReceiver`?